src/foam/core/alarming/scriptparameters.jrl

```diff
@@ -0,0 +1,10 @@
+p({
+  class: "foam.core.script.ScriptParameter",
+  id: "diskSpaceMonitoringCron",
+  name: "diskSpaceMonitoringCron",
+  parameters: {
+    "THRESHOLD_PERCENT": 10,
```

Comment:
The default is 90 and this is 10.  Believe this should be 90. 
```suggestion
    "THRESHOLD_PERCENT": 90,
```

Reply:
oops that was for my testing, thanks